delta/testkit/src/main/scala/ch/epfl/bluebrain/nexus/testkit/NexusSuite.scala

```diff
@@ -0,0 +1,19 @@
+package ch.epfl.bluebrain.nexus.testkit
+
+import munit.FunSuite
+
+class NexusSuite extends FunSuite {
+
+  implicit protected val classLoader: ClassLoader = getClass.getClassLoader
+
+  protected def group(name: String)(thunk: => Unit): Unit = {
+    val countBefore     = munitTestsBuffer.size
+    val _               = thunk
+    val countAfter      = munitTestsBuffer.size
+    val countRegistered = countAfter - countBefore
+    val registered      = munitTestsBuffer.toList.drop(countBefore)
+    (0 until countRegistered).foreach(_ => munitTestsBuffer.remove(countBefore))
+    registered.foreach(t => munitTestsBuffer += t.withName(s"$name - ${t.name}"))
+  }
```

Comment:
I would just want to know that individual tests can still run in intellij easily? As in, click the test name and then run

Reply:
I don't think it's possible. It's also not possible to run a group like this. And to run a single test (from the CLI for instance) you need to specify "{groupName} - {testName}" to run an individual test.

Reply:
I never use this feature maybe with a munit extension it does

Reply:
It's possible to run a single normal munit test, I just need that to still be possible

Reply:
Cf. https://github.com/scalameta/munit/issues/633